lib/zclient.h

```diff
@@ -245,7 +246,8 @@ struct zapi_ipv6
 };
 
 extern int zapi_ipv6_route (u_char cmd, struct zclient *zclient, 
-                     struct prefix_ipv6 *p, struct zapi_ipv6 *api);
+                     struct prefix_ipv6 *p, struct prefix_ipv6 *src_p,
+                     struct zapi_ipv6 *api);
```

Comment:
I don't like the api.  If dst/src is a tuple, let's treat it like one in the code.  Else we are going to get some weird partial api where sometimes we need it and sometimes we don't.  It is going to create confusion and bugs imo